mesh/mesh.hpp

```diff
@@ -433,7 +433,8 @@ class Mesh
 
    void UpdateNURBS();
 
-   void PrintTopo(std::ostream &out, const Array<int> &e_to_k) const;
+   void PrintTopo(std::ostream &out, const Array<int> &e_to_k,
+                  const std::string &comments) const;
```

Comment:
Sorry, I just noticed this: it is better to add a default value for `comments`: ` = ""`.